bears/configfiles/TOMLBear.py

```diff
@@ -0,0 +1,28 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+
+
+@linter(executable='tomlv',
+        output_format='regex',
+        output_regex=r"Error in '(?P<filename>.+)': "
```

Comment:
Collecting `filename` in the regex is deprecated, I believe, as the non-global bears operate on a single file at a time anyway, so it would be a catastrophic failure if the regex-collected filename wasnt the same as the filename given on the command line.